app/models/comment.rb

```diff
@@ -1,6 +1,15 @@
 class Comment < ApplicationRecord
   belongs_to :debate
+  belongs_to :slack_user, class_name: Slack::User, foreign_key: :slack_user_id
```

Comment:
shouldn't it be an optional relation ? I think belongs_to checks presence by default